scripts/in_container/entrypoint_ci.sh

```diff
@@ -170,7 +170,33 @@ set -u
 export RESULT_LOG_FILE="/files/test_result.xml"
 
 if [[ "${GITHUB_ACTIONS}" == "true" ]]; then
-    EXTRA_PYTEST_ARGS=(
+    if [[ "${TEST_TYPE}" == "Helm" ]]; then
+        EXTRA_PYTEST_ARGS=(
```

Comment:
Can you extract common args? I know that Bash and boards aren't your best friends, but I have an example for you that shows how it can be done in Bash. 
https://github.com/apache/airflow/blob/872b1566a11cb73297e657ff325161721b296574/scripts/in_container/run_cli_tool.sh#L72-L84
https://github.com/apache/airflow/blob/872b1566a11cb73297e657ff325161721b296574/scripts/in_container/run_cli_tool.sh#L88-L91
https://github.com/apache/airflow/blob/872b1566a11cb73297e657ff325161721b296574/scripts/in_container/run_cli_tool.sh#L102
https://github.com/apache/airflow/blob/872b1566a11cb73297e657ff325161721b296574/scripts/in_container/run_cli_tool.sh#L138
https://github.com/apache/airflow/blob/872b1566a11cb73297e657ff325161721b296574/scripts/in_container/run_cli_tool.sh#L141-L143
https://github.com/apache/airflow/blob/872b1566a11cb73297e657ff325161721b296574/scripts/in_container/run_cli_tool.sh#L146